target/linux/ath79/dts/ar9341_engenius_eap300-v2.dts

```diff
@@ -1,22 +1,10 @@
 // SPDX-License-Identifier: GPL-2.0-or-later OR MIT
 
-#include "ar9341.dtsi"
-
-#include <dt-bindings/gpio/gpio.h>
-#include <dt-bindings/input/input.h>
-#include <dt-bindings/mtd/partitions/uimage.h>
+#include "ar934x_senao_ap.dtsi"
```

Comment:
So, you just replace ar9341.dtsi with ar9344.dtsi here? Does not sound right to me ...

Reply:
I have a better idea, just wait a moment ...

Reply:
very good catch as always Adrian 👍🏼 

it is only qca955x that is common there...I'll fix this right now

Reply:
I was going to move the `ar934x.dtsi` lines back to the DTS of each device

if you have a better idea I'll wait